deployer/hosts.yml

```
@@ -0,0 +1,102 @@
+.base: &base
+    hostname: guust
+    user: lodewijk
+    application: deploytest
+    deploy_path: /2kdata/dev/____users/lodewijk/deploytest
+    shared_files_path: /2kdata/dev/289_europeana/websites/europeana_files
+    repository: git@github.com:europeana/EuropeanaCopeBolt.git
+    mysql_host: localhost
+    mysql_db: europeana_cope
+    git_tty: true
+    keep_releases: 10
+    cleanup_use_sudo: true
+    snapshots_dir: snapshots
+    shared_files: 
+      - '.bolt.yml'
+    keep_files: 
+      - 'app/config/menu.yml'
+      - 'app/config/taxonomy.yml'
+      - 'app/config/extensions/zohoimport.europeana_local.yml'
+#      - 'extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt'
```

```suggestion
      - 'extensions/bundles/europeana/zohoimport/TokenStorage/zcrm_oauthtokens.txt'
```
Was commented out, but I think we need it